client/src/components/Navbar/Navbar.tsx

```diff
@@ -63,15 +151,18 @@ export default function Navbar({ user, profile, logout }: Props): JSX.Element {
 
   const UserNav = () => (
     <Grid>
-      <Link to="/checkout" className={classes.link}>
-        <Button color="primary" className={`${classes.btn} ${classes.signupbtn}`} variant="contained">
-          Checkout
-        </Button>
-      </Link>
+      {profile && !profile?.isDogSitter && (
+        <>
+          <Button className={classes.userNavItem} onClick={handleBecomeSitter}>
+            Become a Sitter
```

Comment:
You can underline 'Become a Sitter' to make it standout as it is a major change to a user's account. In addition to that, the 'Become a Sitter' button in the Navbar has no function when a user is not logged in, we can either remove it or add a redirect to signup in case user is not logged in?